README.md

```diff
@@ -0,0 +1,35 @@
+Common configurations for Hollowverse repos
+=============================================
+
+This repository contains shared configuration files used across multiple [Hollowverse](https://github.com/hollowverse) repositories.
+
+It currently includes configuration files for:
+
+* TypeScript ([`tsconfig.json`](./tsconfig.json))
+* TSLint ([`tslint.json`](./tslint.json))
+* ESLint ([`eslint.json`](./.eslintrc.json))
+* Stylelint ([`.stylelintrc`](./.stylelintrc))
+
+## How to use the shared configuration
+
+1. Install this package as dependency of your project using the [GitHub URL](https://github.com/hollowverse/common-config).
+     
+     ```bash
+     yarn add https://github.com/hollowverse/common-config --dev
+     # or
+     npm install https://github.com/hollowverse/common-config --save-dev
+     ```
+2. Install the [peer dependencies](./package.json#L7) for each of the tools used in the new project.
+3. Extend your project configuration files with the corresponding files from this package. For example, here is how to extend `tsconfig.json`:
+    
+    ```json
+    {
+      "extends": "./node_modules/common-config/tsconfig.json",
+      "compilerOptions": {
+        // ...
```

Comment:
This looks like this when rendered

![image](https://user-images.githubusercontent.com/26730388/29198712-5381cf08-7dfb-11e7-8352-1cb350f5cbce.png)